packages/internal-tooling/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@stardust-ui/internal-tooling",
+  "version": "0.23.1",
+  "dependencies": {
+    "@babel/core": "^7.3.4",
```

Comment:
could you, please, mention newly added dependencies in the PR's description?

Reply:
Done 👍